app/grants/templates/grants/_new.html

```diff
@@ -39,418 +39,432 @@
         {% include 'grants/nav.html' %}
       </div>
 
-      <div class="container-fluid pb-5 pt-5" id="gc-new-grant">
-        <form action="" @submit="checkForm" ref="formNewGrant" :class="`${Object.keys(errors).length ? 'invalid' : ''}`" v-cloak>
-          <div class="container mt-3 mb-3  position-relative col-lg-7 mx-auto">
-
-            <div class="row p-4 p-md-5">
-
-              <div class="col-12 text-center">
-                <h1 class="font-title-xl mb-4">Create a Grant</h1>
-                <p class="font-bigger-1 font-weight-light mb-0">
-                  We're excited to learn about your project.
-                </p>
-                <p class="text-center">
-                  <button @click="showQuickStart()" type="button" class="font-weight-light font-bigger-1 btn btn-link">
-                    Open Grant Guide
-                  </button>
-                </p>
-              </div>
+      {% if user.is_authenticated %}
```

Comment:
The change to the template included a check for user authentication. The content in the if block is what was there before, no changes, the content in the else block is new and needs review